demos/classification_benchmark_demo/cpp_gapi/README.md

```diff
@@ -161,6 +162,10 @@ For example, use the following command-line command to run the application:
                       -u CDM
 ```
 
+To let the demo find onnx libs copy `onnxruntime_providers_openvino.dll` from `lib` and `onnxruntime.dll`, `onnxruntime_providers_shared.dll` from `bin` dirs  of onnxruntume install to the folder with the demo executable.
```

Comment:
`onnxruntime_providers_openvino.dll` is already mentioned

`DirectML.dll` doesn't exist for me. Are you sure?

Reply:
`DirectML.dll` should be in binary folder if you built `DML` execution provider.

Reply:
Well, in fact `DirectML.dll` is also part of `C:\system32` but if it's custom it also should be placed near the binary

Reply:
I suppose whether anyone has proficient knowledge how to build the custom `DirectML.dll` instead of using the default one then this experimentator is well informed how to deploy it. In my perspective, this comprehensive instructions are redundant

Reply:
The instructions are mostly for me in first place. Usually I forget such specifics in two weeks.